test/test_rpc.py

```diff
@@ -3,21 +3,22 @@
 
 import sys
 import unittest
+from os import getenv
 
 import torch
 import torch.distributed as dist
+from common_distributed import MultiProcessTestCase
+from common_utils import load_tests, run_tests
+from torch.distributed.rpc import RpcBackend
```

Comment:
This section should be put after the if check, otherwise the import will fail. This is currently blocking the Windows builds.

Reply:
@peterjc123 thanks for pointing this out, I am putting up a diff now.

Reply:
@peterjc123 https://github.com/pytorch/pytorch/pull/27126/files

Reply:
Fixed in 27126.